libs/coin-evm/src/api/rpc/rpc.common.ts

```diff
@@ -130,6 +130,15 @@ export const getGasEstimation = (
   transaction: EvmTransaction,
 ): Promise<BigNumber> =>
   withApi(account.currency, async api => {
+    /**
+     * NOTE: either we do this here, or we duplicate this logic in all functions
+     * using getGasEstimation (in order not to update the tx gasLimit when in
+     * "custom" mode)
+     */
+    if (transaction.feesStrategy === "custom") {
+      return transaction.gasLimit;
+    }
+
```

Comment:
Ok so you made this for the `gasLimit`. Let's discuss this when you come back so we can assess the differences between the 2 strategies of `useGasLimit` vs custom mode, but I think the custom mode has more drawbacks